$NetBSD: patch-aa,v 1.1 2006/11/27 14:24:01 dsainty Exp $

Under NetBSD, NUM_FIELDS has whitespace padding - courtesy of "wc -w".  Use
a numeric comparison instead of a string comparison to ignore the differing
whitespace.

--- sarab.sh.orig	2006-11-28 03:04:14.000000000 +1300
+++ sarab.sh	2006-11-28 03:05:14.000000000 +1300
@@ -169,7 +169,7 @@
 fi
 
 ## Two fields means this is a diffential/incremental backup, one field means this is a full backup   
-if [ "$NUM_FIELDS" = "2" ]; then
+if [ "$NUM_FIELDS" -eq "2" ]; then
    # First field is the current archive
    CURRENT_ARCHIVE=$(echo $CURRENT_LINE | cut -f 1 -d" ")
    # Second field is the reference archive
